initial guide steps (draft)

adinsmoor opened the pull request:
Adding first few steps to verify preview and formatting

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If related to an open 
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
<!--
Uncomment when publishing docs for a prerelease version of dbt:
- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)
-->
- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

adinsmoor committed:
initial guide steps (draft)

Adding first few steps to verify preview and formatting

adinsmoor committed:
add last few sections, comment out images

adinsmoor committed:
add image files (still unlinked)

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

Commit:
fix error

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,563 @@
+id: "mesh-qs"
+level: 'Intermediate'
+icon: 'guides'
+tags: ['dbt Cloud','Quickstart']
```
replaced intermediate w quickstart here since we have intermediate above. otherwise intermediate was showing up twice in the doc.

adinsmoor replied:
sounds good!

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,563 @@
+
+<!-- <Lightbox src="/img/guides/dbt-mesh/deprecation_date_warning.png" title="Deprecation date warning" /> -->
+
+## View lineage with dbt Explorer
```
i updated this header here to include the product name and action. i think otherwise someone might not make the connection that 'explore' in the cloud UI is that same as 'db explorer'. i think that was a bigger discussion in general but made that small tweak - hope that's ok!

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,563 @@
+<!-- <Lightbox src="/img/guides/dbt-mesh/deprecation_date_warning.png" title="Deprecation date warning" /> -->
+
+## View lineage with dbt Explorer
+
```
if you'd like, it might be mice to wrap the last bits in a confetti component for a nice little easter egg! we have one in the core to cloud guide ([see here and click anywhere on the page](https://docs.getdbt.com/guides/core-to-cloud-1?step=10)).

to do this, just wrap the content inside the `<ConfettiTrigger>..your content..</ConfettiTrigger>`  component!

adinsmoor replied:
love it!

adinsmoor committed:
add images, adjust format

- add image lightboxes back in
- add admonitions back in
- adjust formatting (bold format button names, single quote file names)

adinsmoor committed:
adjust title

adinsmoor committed:
fix images

adinsmoor committed:
adjust admonition title

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

Commit:
azzam's feedback

Commit:
mirna's feedback

Commit:
updates

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,585 @@
+## Set up a foundational project
+
+A foundation project is where 
+
```
@adinsmoor can i confirm if this is right?

adinsmoor replied:
I'll tweak this to upstream project

Commit:
formatting

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,575 @@
+
+In the previous section, you've arranged your basic building blocks, now let's integrate dbt Mesh. Although the Finance team requires the `fct_orders` model for analyzing payment trends, other models, particularly those in the staging layer used for data cleansing and joining, are not needed by downstream teams.
+
+To make `fct_orders` publicly available, add an `access: public` clause to the relevant YAML file. Add and save the following file to the project:
```
this means they have to create a 'models/core/core.yml' file in the Jaffle | Data Analytics project, right?

adinsmoor replied:
yup! in line 211 I noted they'll add and save the 'models/core/core.yml' file

Commit:
updates

mirnawong1 commented on website/docs/guides/mesh-qs.md:
````diff
@@ -0,0 +1,575 @@
+select * from {{ ref('fct_orders', v=2) }}
+select * from {{ ref('fct_orders') }}
+```
+
````
what should they see @adinsmoor?

adinsmoor replied:
I'll add another sentence to clarify what they should see: "Note how each ref is compiled to the specified version (or the latest, when not specified)."

Commit:
updates

Commit:
updates

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,600 @@
+
+## View lineage with dbt Explorer <Lifecycle status="enterprise"/>
+
+Use dbt Explorer to view the lineage across projects in dbt Cloud. Navigate to the **Explore** page for each of your projects &mdash; you should now view the [lineage seamlessly across projects](/docs/collaborate/explore-multiple-projects).
```
coul we have an image here?

Commit:
final tweaks

Commit:
add links

Commit:
tweak

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,616 @@
+
+dbt Mesh, available in [dbt Cloud Enterprise](https://www.getdbt.com/), is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections &mdash; for faster data development. 
+
+In this guide you’ll learn how to set up a multi-project design using foundational concepts of [dbt Mesh](https://www.getdbt.com/blog/what-is-data-mesh-the-definition-and-importance-of-data-mesh). In this guide, you’ll:
```
```suggestion
This guide will teach you how to set a multi-project design using foundational concepts of [dbt Mesh](https://www.getdbt.com/blog/what-is-data-mesh-the-definition-and-importance-of-data-mesh) and how to implement a data mesh in dbt Cloud:
```

mirnawong1 commented on website/docs/guides/mesh-qs.md:
```diff
@@ -0,0 +1,616 @@
+- Add model access, versions, and contracts
+- Set up a dbt Cloud job that is triggered on completion of an upstream job
+ 
+This guide explains how to implement a data mesh in dbt Cloud. For more information on why data mesh is important, read this post: [What is data mesh? The definition and importance of data mesh](https://www.getdbt.com/blog/what-is-data-mesh-the-definition-and-importance-of-data-mesh).
```
```suggestion
For more information on why data mesh is important, read this post: [What is data mesh? The definition and importance of data mesh](https://www.getdbt.com/blog/what-is-data-mesh-the-definition-and-importance-of-data-mesh).
```

mirnawong1 committed:
Update mesh-qs.md

mirnawong1 committed:
Update mesh-qs.md

mirnawong1 committed:
Update mesh-qs.md

adinsmoor committed:
add final Explorer image and adjust a couple phrases

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

mirnawong1 committed:
Update mesh-qs.md

mirnawong1 committed:
Update mesh-qs.md

mirnawong1 committed:
Update mesh-qs.md

Commit:
Merge branch 'feature/mesh-quickstart' of https://github.com/dbt-labs/docs.getdbt.com into feature/mesh-quickstart

Commit:
tweak

Commit:
final tweaks

Commit:
add link

mirnawong1 reviewed (approved):
this looks fantastic! thank you so much for opening this pr up @adinsmoor, we really appreciate it! ✨ ready to ship!

mirnawong1 committed:
Merge branch 'current' into feature/mesh-quickstart

mirnawong1 committed:
Update mesh-qs.md
